src/JsonUtil.cpp

```diff
@@ -53,7 +53,7 @@ namespace BrainCloud
 
     Json::Value JsonUtil::stringVectorToJson(const std::vector<std::string> & in_stringVec)
     {
-        Json::Value value;
+        Json::Value value(Json::arrayValue);
```

Comment:
This was wrong already, could have caused issues in other places! returned `null` json value if array was empty, instead of an empty array.